docs/source/usersguide/input.rst

```diff
@@ -725,10 +725,12 @@ a material default temperature.
 ``<temperature_method>`` Element
 --------------------------------
 
-The ``<temperature_method>`` element has an accepted value of "nearest" or
-"interpolation". A value of "nearest" indicates that for each cell, the nearest
-temperature at which cross sections are given is to be applied, within a given
-tolerance (see :ref:`temperature_tolerance`). A value of "multipole" indicates
+The ``<temperature_method>`` element has an accepted value of "nearest",
+"interpolation", or "multipole". A value of "nearest" indicates that for each
+cell, the nearest temperature at which cross sections are given is to be
+applied, within a given tolerance (see :ref:`temperature_tolerance`). A value of
+"interpolation" indicates that cross sections are to be interpolated between
```

Comment:
This has been updated. I also added a section in our Methods manual talking about temperature treatments.